api/src/gcnotfiy-services/gcnotify-service.ts

```diff
@@ -0,0 +1,97 @@
+import axios from 'axios';
+import { ApiError, ApiErrorType } from '../errors/custom-error';
+import { IgcNotfiyPostReturn, IgcNotfiyGenericMessage } from '../models/gcnotify';
+
+export class GCNotifyService {
+  /**
+   * Sends api call to gcnotify to send notification
+   *
+   *
+   * @param {string} url
+   * @param {object} config
+   * @param {object} data
+   */
+  async sendGCNotification(url: string, config: object, data: object): Promise<IgcNotfiyPostReturn> {
+    const response = await axios.post(url, data, config);
+
+    const result = (response && response.data) || null;
+
+    if (!result) {
+      throw new ApiError(ApiErrorType.UNKNOWN, 'Failed to send Notification');
+    }
+
+    return result;
+  }
+
+  /**
+   * Send email notification to recipient
+   *
+   *
+   * @param {string} emailAddress
+   * @param {object} config
+   * @param {object} message
```

Comment:
nit: some of these jsdocs are a bit out of date. They don't list the latest param types or return types, etc.